source/common/runtime/runtime_features.cc

```diff
@@ -80,6 +80,7 @@ RUNTIME_GUARD(envoy_restart_features_explicit_wildcard_resource);
 RUNTIME_GUARD(envoy_restart_features_remove_runtime_singleton);
 RUNTIME_GUARD(envoy_restart_features_udp_read_normalize_addresses);
 RUNTIME_GUARD(envoy_restart_features_use_apple_api_for_dns_lookups);
+RUNTIME_GUARD(envoy_reloadable_features_lambda_sanitize_host_header);
```

Comment:
Also it is preferred to record this change under change log https://github.com/envoyproxy/envoy/blob/main/changelogs/current.yaml#L45C5-L45C5 under minor behavior change.